README.md

```
@@ -50,20 +50,103 @@ It powers transparent and auditable sustainability tracking within the **Stabili
+- **Verification System**: Community-driven verification with proof submission
+- **Incentive Tokens**: Multiple ERC20 tokens rewarding different ecosystem activities
+- **User Profiles**: On-chain user registration and reputation tracking
+
+---
+
+## Architecture
 
-### Contracts
+### Core Components
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)